### PR TITLE
[MP-23]Navigation section component

### PR DESCRIPTION
**Pull request by thisisofficial**
PR checklist

### Description

Added a navigation component that can be edited with simple props
image

### Testing Checklist

- [ ] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [MP-23](https://cd95.atlassian.net/browse/MP-23)

### CODE REVIEW

**Comment by thisisofficial** [comment]
@caarlosdamian review please

**Comment by thisisofficial** [comment]
Review request bump @caarlosdamian please

File: src/components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    transition: .2s;
+    flex-basis: 100%;
+
+}
+
+.navigation-button:hover{
+    cursor: pointer;
+    opacity: 1;
+    font-weight: 500;
+
```
Comment (caarlosdamian) [inline src/components/navigation/navigation.css:38]: remove space

**Review by caarlosdamian — CHANGES_REQUESTED** [review]
some comments

**Review by caarlosdamian — APPROVED** [review]

**Pull request merged**